USE_CASES.md

```diff
@@ -1344,6 +1345,129 @@ func main() {
 }
 ```
 
+<a name="tutorial_deployments"></a>
+# Tutorial Deployments
+## Heroku
+
+### Require
+* [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)
+* [godep](https://github.com/tools/godep)
+
+### Initial project with example code
+
+```go
+package main
+
+import (
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sendgrid/rest"
+	sendgrid "github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+var sendgridkey = "SENDGRID_API_KEY"
```

Comment:
Could you get the api key from an environment variable like you did with the port below. 
```golang
sendgridAPIKey := os.Getenv("SENDGRID_API_KEY")
```